build.rs

```diff
@@ -28,6 +28,8 @@ fn get_config() -> PathBuf {
 fn print_config() {
     println!("cargo:rustc-link-lib=static=winbridge");
     println!("cargo:rustc-link-lib=dylib=user32");
+    println!("cargo:rustc-link-lib=dylib=gdiplus");
+    println!("cargo:rustc-link-lib=dylib=stdc++");
```

Comment:
please wait merge. I don't understand how to make condition for the gcc compiler.